lib/rubocop/cop/style/multiline_ternary_operator.rb

```diff
@@ -4,6 +4,19 @@ module RuboCop
   module Cop
     module Style
       # This cop checks for multi-line ternary op expressions.
+      #
+      # @example
+      #   # bad
+      #   a = cond ?
+      #     b : c
+      #   a = cond ? b :
+      #       c
+      #   a = cond ?
+      #       b :
+      #       c
+      #
+      #   # good
+      #   a = cond ? b : c
```

Comment:
Mutiline `if-else`end` is also good, and the cop says "use `if` or `unless` instead.". So I think the example should contain `if-else-end` case.



```
a =
  if cond
    b
  else
    c
  end
```